include/triton/Dialect/TritonGPU/Transforms/DecomposeScaledBlocked.h

```diff
@@ -32,6 +32,11 @@ class DecomposeScaledBlocked : public OpRewritePattern<DotScaledOp> {
   TypedValue<RankedTensorType>
   cvtDotOperand(PatternRewriter &rewriter, DotScaledOp scaledDotOp, int opIdx,
                 TypedValue<RankedTensorType> v) const;
+  TypedValue<RankedTensorType>
+  extendAndBroadcastScale(PatternRewriter &rewriter, DotScaledOp scaledDotOp,
+                          TypedValue<RankedTensorType> &scale,
```

Comment:
No need to have the `&` here given `TypedValue<RankedTensorType>` itself is pretty much just a pointer.